stock_reserve/test/stock_reserve.yml

```diff
@@ -6,7 +6,7 @@
     # Travis, the hook performing this operation for the new stock reservation
     # location is run after the test execution. This causes the stock level
     # computation to be wrong at the time the tests are run.
-    self._parent_store_compute(cr)
+    self._parent_store_compute()
```

Comment:
I agree. Although this PR focuses on migration and not improving/refactoring the module. My suggestion would be to push it through and improve it later on.